packages/frontend/package.json

```diff
@@ -50,6 +50,7 @@
     "reselect": "^4.0.0",
     "semantic-ui-css": "^2.4.1",
     "semantic-ui-react": "^0.88.1",
+    "sentry": "^0.1.2",
```

Comment:
@danielwpz I don't think you need to add sentry here?

Reply:
Please 🔪 this :)